validator/src/main.rs

```diff
@@ -1210,6 +1210,11 @@ pub fn main() {
                 .takes_value(false)
                 .help("Use QUIC to send transactions."),
         )
+        .arg(
+            Arg::with_name("enable_quic_servers")
+                .hidden(true)
```

Comment:
should this arg conflict with `tpu_use_quic`, or doesn't matter?

Reply:
don't think it matters, they're independent resources (locally)